test/init.js

```diff
@@ -147,7 +147,7 @@ describe('#init', () => {
     assert.deepEqual(statsd.globalTags, ['gtag', 'dd.internal.entity_id:04652bb7-19b7-11e9-9cc6-42010a9c016d']);
   });
 
-  it('should attempt to cache a dns record if dnsCache is specified', done => {
+  it('should lookup a dns record if dnsCache is specified', done => {
```

Comment:
Make clear this isn't really testing the attempt to cache, just the lookup.  I tried to add a test for this bug, but I didn't see a straightforward way without mocking in place.  Making a note in https://github.com/brightcove/hot-shots/issues/63